File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1329.cpp
#include <iostream>

using namespace std;

int main(){
    int n, v;
    
    while(true){
        cin >> n;
        if(n == 0) break;
        int maria = 0, joao = 0;
        
        for(int i = 0; i < n; i++){
            cin >> v;
            if(v == 0) maria++;
            else joao++;
        }
        
        cout << "Mary won " << maria << " times and John won " << joao << " times" << endl;
    }
    
    return 0;
}

File: codesLeftInWindows/CP-Problems-tilnoeneBackup/URI/1329.py
while True:
    n = int(input())
    if n == 0: break;
    
    maria = joao = 0
    
    v = input().split()
    
    for i in range(n):
        k = int(v[i])
        if(k == 0): maria += 1
        else: joao += 1
        
    print("Mary won {} times and John won {} times".format(maria, joao))